pages/ag/neuro-rse.md

```diff
@@ -0,0 +1,111 @@
+
+To join, contact one of the coordinators on the US-RSE slack (see the [Questions?](#questions) section below).
+
+# Activities
+
+Neuro-RSE meets virtually monthly, with alternating times (one more East Coast-friendly, one more West Coast-friendly). Additional in-person meetings at RSE or Neuroscience events may also be organized.
+
+During our meetings, we have brief lightning talks, to share our work with each other, as well as discussions on relevant topics. These topics are determined by group members and may include: how are RSEs managed and incentivized in neuroscience, how to convince scientists of the value of our work (and the time required), what does day-to-day work look like, how to incentivize building / using common resources instead of reinventing the wheel, how to formally run teams and open source collaborations.
+
+We have a Slack channel in the US-RSE space for discussions.
+
+# Questions?
+
+Reach out to the Neuro-RSE coordinators Billy Broderick or David Nicholson on the [US-RSE slack](https://us-rse.org/join/) with questions.
+
+# Group coordinators
+
+- [Billy Broderick](wfbroderick.com/), data scientist at [Flatiron Institute's Center For Computational Neuroscience](https://www.simonsfoundation.org/flatiron/center-for-computational-neuroscience/) 
+- [David Nicholson](https://nicholdav.info/), lead developer of the [VocalPy software community](http://www.vocalpy.org/)
+
+# Active members
+
+Active members attend our regular meetings at least once a quarter (once every 3 meetings).
+
+- [Lezlie España](https://www.linkedin.com/in/lezlieespana), Programmer Analyst in Brain Injury Research at the Medical College of Wisconsin
+- [Alessandro Felder](https://github.com/alessandrofelder), Senior Research Software Engineer at the [Sainsbury Wellcome Centre's Neuroinformatics Unit](https://neuroinformatics.dev/), technical lead of the [BrainGlobe Initiative](https://brainglobe.info/)
+- [Yaroslav O. Halchenko](https://github.com/yarikoptic), Research Associate Professor at the [Dartmouth College](https://dartmouth.edu), leading the [Center for Open Neuroscience](https://centerforopenneuroscience.org) at the [Department of Psychological and Brain Sciences](https://pbs.dartmouth.edu) leading projects such as [NeuroDebian](https://neuro.debian.net/), [DataLad](https://datalad.org), [DANDI](https://dandiarchive.org) and others
+- [Ryan Ly](https://www.linkedin.com/in/ryanly), scientific data engineer at Lawrence Berkeley National Lab
+- [Daniel McCloy](https://dan.mccloy.info), Research Scientist at Institute for Learning & Brain Sciences, University of Washington
+- [Tab Memmott](https://www.linkedin.com/in/tab-memmott/) (he/him), Lead Research Software Engineer with the [Consortium for Accessible Multimodal Brain-Body Interfaces](https://www.cambi.tech/)
+- [Adam Tyson](https://adamltyson.com/), Head Research Engineer at the [Sainsbury Wellcome Centre's Neuroinformatics Unit](https://neuroinformatics.dev/)
```

Comment:
now that coordinators are above -- let's just sort members?

```suggestion
- [Adam Tyson](https://adamltyson.com/), Head Research Engineer at the [Sainsbury Wellcome Centre's Neuroinformatics Unit](https://neuroinformatics.dev/)
- [Alessandro Felder](https://github.com/alessandrofelder), Senior Research Software Engineer at the [Sainsbury Wellcome Centre's Neuroinformatics Unit](https://neuroinformatics.dev/), technical lead of the [BrainGlobe Initiative](https://brainglobe.info/)
- [Daniel McCloy](https://dan.mccloy.info), Research Scientist at Institute for Learning & Brain Sciences, University of Washington
- [Lezlie España](https://www.linkedin.com/in/lezlieespana), Programmer Analyst in Brain Injury Research at the Medical College of Wisconsin
- [Ryan Ly](https://www.linkedin.com/in/ryanly), scientific data engineer at Lawrence Berkeley National Lab
- [Tab Memmott](https://www.linkedin.com/in/tab-memmott/) (he/him), Lead Research Software Engineer with the [Consortium for Accessible Multimodal Brain-Body Interfaces](https://www.cambi.tech/)
- [Yaroslav O. Halchenko](https://github.com/yarikoptic), Research Associate Professor at the [Dartmouth College](https://dartmouth.edu), leading the [Center for Open Neuroscience](https://centerforopenneuroscience.org) at the [Department of Psychological and Brain Sciences](https://pbs.dartmouth.edu) leading projects such as [NeuroDebian](https://neuro.debian.net/), [DataLad](https://datalad.org), [DANDI](https://dandiarchive.org) and others
```

Reply:
oh, I just like the "most obvious" or "easiest" , i.e. was piping into a `sort` on Linux so it went by last first name.  I am ok either way, you decide.

Reply:
Let's keep it with last names for now-- I'm just making a call so we can get this merged in before the next meeting of the group